src/usr/local/www/system_usermanager.php

```diff
@@ -705,7 +721,7 @@ function build_cert_table() {
 		'act',
 		null,
 		'hidden',
-		''
+		$act
```

Comment:
I think this should have been `$act` before, but please have a look if it's right the way it is now.

Reply:
It should be empty as it was.

Reply:
But then there's the problem that the empty POST parameter overrides the non-empty GET parameter, and the form won't show right when you submit but there's an error.